test/jdk/javax/swing/JTable/KeyBoardNavigation.java

```diff
@@ -179,6 +128,7 @@ public void setValue(Object value) {
         frame.add(scrollPane);
         frame.pack();
         frame.setVisible(true);
```

Comment:
no need for frame.setVisible since PassFailJFrame will do that

Reply:
Done.